dashboard/src/pages/Proposal/proposalDetail.tsx

```
@@ -0,0 +1,74 @@
+              symbol: expanded ? '' : <span className={'expanded-text'}>view full text <DownOutlined /></span>,
+            }}
+            className={'content'}>
+            I have heard of your paintings too, well enough; God has given you one face, and you make yourselves another: you jig and amble, and you lisp, and nickname God's creatures, and make your wantonness your ignorance. Go to, I'll no more on't; it hath made me mad. I say, we will have no more marriages: those that are married already, all but one, shall live; the rest shall keep as they are. To a nunnery, go.
+          </Typography.Paragraph>
+        </Flex>
+        <Flex className={'progress-bar-base'} style={{width:'550px'}}>
+          <BottomProgressBar status={3} />
+        </Flex>
+        <Flex vertical>
+          {[1,3,4]?.map((item,index)=> {
+            return showAll  ? <Discussions showLine={showAll && index < [1,2,3].length-1} status={index} /> : (index === 0 ? <Discussions showLine={showAll && index < [1,2,3].length-1} status={index} /> : null);
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)